src/main/java/vazkii/botania/client/gui/lexicon/GuiLexicon.java

```diff
@@ -259,7 +259,7 @@ public final void drawScreen(int x, int y, float partialTicks) {
 	
 	public void drawScreenAfterScale(int par1, int par2, float par3) {
 		float time = ClientTickHandler.ticksInGame + par3;
-		timeDelta = time - lastTime;
+		timeDelta = time - lastTime + partialTicks;
```

Comment:
should this use the new partialTicks (`par3`) instead of the old one?

Reply:
Not sure, but when I tested it and used the result, there was no visible shaking. Will test both tomorrow.

Reply:
They both work fine. Should I update the pull request to use par3 for consistency?

Reply:
yeah, also rename those parameters to have actual names while you're at it :P